app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorRequestView.java

```diff
@@ -470,6 +471,7 @@ private void getFavorFromView() {
     }
     // TODO: Get reward from frontend (currently being set by default to 0)
     Favor favor = new Favor(title, desc, userId, loc, favorStatus, 0);
+    favor.setRequesterNotifId(notifId);
```

Comment:
Should be a parameter of the constructor since this is initialized every time a favor is created from a requester